integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/mediator/test/classMediator/PropertyPersistenceDeletingAndAddingTstCase.java

```diff
@@ -83,29 +91,28 @@ public void testMediationPersistenceDeletingAndAdding() throws Exception {
         Deleting User and email   refer: https://wso2.org/jira/browse/TA-532           param 5
          */
 
+            serverConfigurationManager.removeFromComponentLib(CLASS_JAR_FIVE_PROPERTIES);
+            serverConfigurationManager.copyToComponentLib
+                    (new File(getClass().getResource(JAR_LOCATION + File.separator + CLASS_JAR_FOUR_PROPERTIES).toURI()));
+            loadSampleESBConfiguration(0);
+            /* waiting for the new config file to be written to the disk */
+            Thread.sleep(10000);
+            serverConfigurationManager.restartGracefully();
 
-        serverConfigurationManager.removeFromComponentLib(CLASS_JAR_FIVE_PROPERTIES);
-        serverConfigurationManager.copyToComponentLib
-                (new File(getClass().getResource(JAR_LOCATION + File.separator + CLASS_JAR_FOUR_PROPERTIES).toURI()));
-        loadSampleESBConfiguration(0);
-        /* waiting for the new config file to be written to the disk */
-        Thread.sleep(10000);
-        serverConfigurationManager.restartGracefully();
-
-        super.init();
-        loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/class/class_property_persistence_four_properties.xml");
+            super.init();
+            loadESBConfigurationFromClasspath("/artifacts/ESB/mediatorconfig/class/class_property_persistence_four_properties.xml");
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getMainSequenceURL(),null, "IBM");
+            response = axis2Client.sendSimpleStockQuoteRequest(getMainSequenceURL(), null, "IBM");
```

Comment:
Can you remove these formatting changes? Let's only keep the changes you introduced.

Reply:
Fixed. Please refer latest commit.